src/sql/SqlService.ts

```diff
@@ -128,26 +128,24 @@ import {SqlPage} from './SqlPage';
  */
 export interface SqlService {
     /**
-     * Executes SQL and returns an SqlResult.
+     * Executes SQL and returns an {@link SqlResult}.
```

Comment:
` * let result = client.getSqlService().execute('SELECT * FROM person');` above doc must be updated